src/emulator/storage/crc.ts

```diff
@@ -43,12 +45,14 @@ export function crc32c(bytes: Buffer): number {
 
 /**
  * Adapted from:
- *  - https://github.com/googleapis/nodejs-storage/blob/0c1fa3934a52a608366a8c6c798c43516dd03dbf/src/file.ts#L1406-L1409
+ *  - https://github.com/googleapis/nodejs-storage/blob/1d7d075b82fd24ea3c214bd304cefe4ba5d8be5c/src/crc32c.ts
```

Comment:
Should we just link the main branch?

Reply:
This permalink keeps track of the file exactly as referenced (for context purposes) - this can help future debugging; like the link that was removed helped me trace where the code was copied from + date/time